database/migrations/2023_05_26_060433_create_post__views_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use App\Models\Post;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('post__views', function (Blueprint $table) {
+            $table->foreignIdFor(Post::class);
```

Comment:
`post_id`, `user_agent` and `ip_address` should be primary keys. Same for `post_likes` table.